config/propel.php

```diff
@@ -63,8 +63,13 @@ function($item)
                         'settings' => [
                             'charset' => $item['charset'],
                             #Array of queries to run when the database connection is initialized
-                            'query' => [
-                                'SET NAMES utf8 COLLATE utf8_unicode_ci, COLLATION_CONNECTION = utf8_unicode_ci, COLLATION_DATABASE = utf8_unicode_ci, COLLATION_SERVER = utf8_unicode_ci'
+                            'query' => [ call_user_func(function() use ($item) {
+                                    if ($item['driver'] == 'mysql') {
+                                        return 'SET NAMES utf8 COLLATE utf8_unicode_ci, COLLATION_CONNECTION = utf8_unicode_ci, COLLATION_DATABASE = utf8_unicode_ci, COLLATION_SERVER = utf8_unicode_ci';
+                                    } elseif ($item['driver'] == 'pgsql') {
```

Comment:
Use [identical comparison](http://php.net/manual/en/language.operators.comparison.php) and [Yoda conditions](https://en.wikipedia.org/wiki/Yoda_conditions) please. Just «push -f» commit to your repo and pr will updates. And i will merge it.